src/bundle/Resources/public/js/scripts/helpers/modal.helper.js

```diff
@@ -0,0 +1,12 @@
+const controlZIndex = (container) => {
```

Comment:
as we discussed on priv, that's not that good of a name, "control z-index" is quite vague and tells nothing (so we'll be lost in half a year), it's more like "force show modal over container", but that's not good name either...